source/developer/slash-commands.rst

```diff
@@ -153,6 +153,17 @@ If the parameter is present, the user is redirected to the specified URL in a ne
 
 The parameter supports any custom protocol including ``http://``, ``https://``, ``ftp://``, ``ssh://`` and ``mailto://``.
 
+Delayed and multiple responses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use the ``response_url`` parameter to supply multiple responses or a delayed response to a slash command. Response URLs can be used to send five additional messages within a 30-minute time period from the original command invocation. 
+
+Delayed responses are useful when the action takes more than three seconds to perform, for instance:
```

Comment:
This comma trips me up, and while I _think_ there may be a way to parse this as correct grammar, it looks like a comma splice. I think the comma would be better replaced with a period or semicolon.